src/app/page.js

```diff
@@ -11,17 +11,18 @@ import RolesHome from "./components/RolesHome";
 const Home = () => {
   return (
     <section id="base-background">
-      <div className="section section-home page-main-side-padding">
+      <div className="section section-home page-main-side-padding ">
         <div className="flex flex-wrap items-center justify-center my-[2rem]">
           <div className="md:flex-1 text-center sm:text-left">
-            <h2 className="font-semibold text-[2.5rem] md:text-[3.5rem] lg:text-[4rem]">
+            <h2 className="font-semibold text-[2rem] md:text-[2.5rem] lg:text-[3rem] text-MidnightBlue leading-10">
               Hey UGA!
             </h2>
-            <h1 className="font-extrabold text-[2.5rem] md:text-[3.5rem] lg:text-[4rem]">
+
+            <h1 className="inline font-black text-[3.5rem] md:text-[4rem] lg:text-[4.5rem] text-MidnightBlue">
               We&#39;re{" "}
```

Comment:
Can you modify the "We're DevDogs," to be in one line, similar to how it is in the Figma? There seems to be a larger gap in between the logo and the title.
Figma:
![image](https://github.com/user-attachments/assets/dbde4fe1-45ce-46a9-89db-c8a523e08014)

The current version with the font modifications is:
![image](https://github.com/user-attachments/assets/dafb1670-3180-4110-a1d1-da2de0290053)